lib/matplotlib/cbook.py

```diff
@@ -783,8 +783,12 @@ def is_sequence_of_strings(obj):
     """
     if not iterable(obj):
         return False
-    if is_string_like(obj):
-        return False
+    if is_string_like(obj) and not isinstance(obj, np.ndarray):
```

Comment:
`is_string_like(obj)` checks if string concatenation is possible. i.e If you can add a string to it, then it is "string like".